pkg/standalone-utils/contracts/relayer/BalancerRelayer.sol

```diff
@@ -88,6 +91,18 @@ contract BalancerRelayer is IBalancerRelayer, Version, ReentrancyGuard {
         _refundETH();
     }
 
+    function vaultActionsQueryMulticall(bytes[] calldata data)
+        external
+        override
+        nonReentrant
+        returns (bytes[] memory results)
+    {
+        results = new bytes[](data.length);
+        for (uint256 i = 0; i < data.length; i++) {
+            results[i] = _queryLibrary.functionDelegateCall(data[i]);
```

Comment:
```suggestion
        uint256 numData = data.length;
        results = new bytes[](numData);
        for (uint256 i = 0; i < numData; i++) {
            results[i] = _queryLibrary.functionDelegateCall(data[i]);
```

Reply:
Sure; the compiler would probably already optimize this, but it saves 7 bytes :)